docs/source/app-dev/app-arch.rst

```diff
@@ -1,295 +1,121 @@
+``create-daml-app`` is a small, but fully functional demo application implementing the recommended
+architecture, including
 
--  Event-driven - applications base their actions on individual ledger events only.
--  State-driven - applications base their actions on some model of all contracts active on the ledger.
+- DAML React libraries
+- quick iteration against the :ref:`DAML Ledger Sandbox <sandbox-manual>`.
+- authentication
+- deploy your application in the cloud as a Docker container
 
-Event-driven applications
-=========================
+It provides an excellent starting point to iterate on, so you can build your own application.
 
-**Event-driven** applications react to events on the the ledger and generate commands and other outputs on a per-event basis. They do not require access to ledger state beyond the event they are reacting to.
 
-Examples are sink applications that read the ledger and dump events to an external store (e.g. an external (reporting) database).
+Frontend
+~~~~~~~~
 
-State-driven applications
-=========================
+We recommended building your frontend with the `React <https://reactjs.org>`_ framework. We provide
+several support libraries to build your React frontend for a DAML application.
 
-**State-driven** applications build up a real-time view of the ledger state by reading events and recording contract create and archive events. They then generate commands based on a given state, not just single events.
++--------------------------------------------------------------+--------------------------------------------------------------------------+
+| Name                                                         | Summary                                                                  |
++==============================================================+==========================================================================+
+| `@daml/react <https://www.npmjs.com/package/@daml/react>`_   | React hooks to query/create/exercise DAML contracts                      |
++--------------------------------------------------------------+--------------------------------------------------------------------------+
+| `@daml/ledger <https://www.npmjs.com/package/@daml/ledger>`_ | DAML ledger object to connect and directly submit commands to the ledger |
++--------------------------------------------------------------+--------------------------------------------------------------------------+
+| `@daml/types  <https://www.npmjs.com/package/@daml/types>`_  | DAML data types                                                          |
++--------------------------------------------------------------+--------------------------------------------------------------------------+
 
-Examples of these are automation and interactive applications that let a user or code react to complex state on the ledger (e.g. the DA Navigator tool).
+You can install any of these libraries by running ``yarn add @daml/react`` in the ``ui`` directory
```

Comment:
I would need to run a separate command for each library, right? So yarn add @daml/react, yarn add @daml/ledger and yarn add @daml/types. If that's the case then maybe write something like

"You can install these libraries by running the following command in the ``ui``directory of your project ``yarn add @``and the name of the library (e.g., ``yarn add @daml/react`` for installing the  ``@daml/react``library)"

Reply:
changed.